docs/designers-developers/developers/themes/theme-json.md

```diff
@@ -0,0 +1,219 @@
+# Themes & Block Editor: Experimental theme.json
+
```

Comment:
Actually, we should add a sentence here to clarify the experimental state of the APIs (like the block-based theme doc)

Reply:
How about https://github.com/WordPress/gutenberg/pull/22518/commits/7f82de86a5a5cc6a9f71675dc5c55855e9e24075 ? Mostly copied verbatim from the block-themes page.

Reply:
Great but I have the same question as https://github.com/WordPress/gutenberg/pull/22518/files#r428571002

Reply:
Is that the link you intended to share? It points to this same thread so I'm not sure I follow.

Reply:
yes, it's the comment bout the FSE requirement :)